### PR TITLE
[NG] Profile Updates

### PR DESCRIPTION
**Pull request by Jashpatel1**
**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](https://github.com/layer5io/layer5/blob/master/CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [ ] Yes, I signed my commits.

### CODE REVIEW

File: src/collections/members/dhruv-patel/index.mdx [hunk 1]
```diff
@@ -2,10 +2,15 @@
 name: Dhruv Patel
 position: Contributor
 image_path: ./dhruv-patel.jpeg
-twitter: DhruvPatel0008
 github: dhruv0000
+twitter: DhruvPatel0008
```
Comment (dhruv0000) [inline src/collections/members/dhruv-patel/index.mdx:6]: Slight update. Its `realDhruvPatel`

**Review by kalradev — DISMISSED** [review]

File: src/collections/members/efereyan-karen/index.mdx [hunk 2]
```diff
@@ -2,10 +2,13 @@
 name: Efereyan Karen
 position: Contributor
 image_path: ./efereyan-karen.jpeg
-twitter: EfereyanK
 github: KarenEfereyan
+twitter: EfereyanK
 linkedin: karen-efereyan-6918aa12b
 blurb: Mesh Head
 status: Active
+badges:
```
Comment (Nikhil-Ladha) [inline src/collections/members/efereyan-karen/index.mdx:10]: No need of adding the frontmatter if the field is empty.

**Comment by Nikhil-Ladha** [comment]
@Tanuj22 the netlify builds are failing with error message related to your recent PR for categories and tags update. Please take a look.
For the time being I have done a rebuild, let's see if it goes well.

**Comment by Tanuj22** [comment]
@Jashpatel1 try and merge `layer5-ng` into this branch once

**Comment by KarenEfereyan** [comment]
Hey @Jashpatel1 well done. My LinkedIn username is now just karenefereyan

**Review by leecalcote — DISMISSED** [review]

**Review by Tanuj22 — APPROVED** [review]

**Pull request merged**